types/errors.go

```diff
@@ -268,3 +275,25 @@ func (err *sdkError) QueryResult() abci.ResponseQuery {
 		Log:  err.ABCILog(),
 	}
 }
+
+func parseCmnError(err string) string {
```

Comment:
This seems like a suboptimal way to parse this, we could have just had a better string method on cmn.error

Reply:
Will open a separate PR for tendermint as well